test/matchMedia.mock.ts

```diff
@@ -0,0 +1,11 @@
+Object.defineProperty(window, 'matchMedia', {
+    writable: true,
+    value: jest.fn().mockImplementation(query => ({
```

Comment:
I needed this because viewport store uses `window.matchMedia`.

Reply:
See comment below.